src/providers/data/blockfrost-provider.ts

```diff
@@ -140,9 +128,9 @@ export class BlockfrostProvider extends BaseDataProvider {
             }
 
             return results;
-        }).catch(() => {
-            return Promise.resolve([]);
+        }).catch((e) => {
```

Comment:
I'd just remove this catch here, since it's basically doing the same thing as before.